shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -312,11 +313,21 @@ public final WindowInsets onApplyWindowInsets(@NonNull WindowInsets insets) {
     viewportMetrics.viewInsetBottom = insets.getSystemWindowInsetBottom();
     viewportMetrics.viewInsetLeft = 0;
 
+    if (Build.VERSION.SDK_INT >= 29) {
```

Comment:
Please use the named constant rather than `29`. There's another property within Build that provides it. You should be able to find an example somewhere else in the Android embedding if you look.

Reply:
I remember that this caused an issue with the pre-submit checks for a reason I did not really understand. I updated it to use the named constant and will let you know if the problem resurfaces.

Reply:
This has been resolved by suppressing the lint and adding a comment for how the code properly guards for the right SDK version.